### PR TITLE
Table paginator

### PR DESCRIPTION
**Pull request by anandtiwary**
## Description
Please include a summary of the change, motivation and context.
- Added pagination styling changes
- Added pagination floating behavior (center aligned and shows up at the bottom of the row). This behavior is controlled via an input property `showFloatingPaginator`. 

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

### Checklist:
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: projects/components/src/table/table.component.scss [hunk 1]
```diff
@@ -140,7 +140,7 @@ $header-height: 32px;
 }
 
 .pagination-controls {
-  position: fixed;
+  // position: fixed;
```
Comment (jake-bassett) [inline projects/components/src/table/table.component.scss:143]: Commented code (I know still draft, so you probably planned on removing)

File: projects/components/src/button/button.component.scss [hunk 2]
```diff
@@ -79,7 +79,7 @@
 
 .bordered {
   &.primary {
-    @include button-style(inherit, $blue-1, $gray-7, $gray-7, $blue-5);
+    @include button-style(inherit, $blue-1, $blue-5, $blue-4, $blue-5);
```
Comment (palbizu) [inline projects/components/src/button/button.component.scss:82]: nit: can you run lighthouse to see if this change supports accessibility?

**Review by palbizu — COMMENTED** [review]
LGTM. Can you check accessibility before merging it?

**Review by palbizu — COMMENTED** [review]
LGTM. Can you check accessibility before merging it?

**Comment by anandtiwary** [comment]
good point. Let me check

File: projects/components/src/button/button.component.scss (hunk 2 above)
Comment (anandtiwary) [reply projects/components/src/button/button.component.scss:82]: Screenshot 2023-05-10 at 5 38 51 PM
Looks good

**Comment by anandtiwary** [comment]
Screenshot 2023-05-10 at 5 38 51 PM

**Review by palbizu — APPROVED** [review]

**Review by Exac — APPROVED** [review]

**Review by jake-bassett — APPROVED** [review]

**Pull request merged**